budget_proj/budget_app/models.py

```diff
@@ -34,6 +42,7 @@ class BudgetHistory(models.Model):
     functional_area_name = models.CharField(max_length=255, default='')
     accounting_object_name = models.CharField(max_length=255, default='')
     service_area_code = models.CharField(max_length=32, default='')
+    service_area = models.ForeignKey(LookupCode)
```

Comment:
Yes, as I mentioned in my comment, "it [the migration] does not initialize the data correctly". I got around that by writing a SQL `UPDATE` statement to initialize the values of the `budget_app_budgethistory.service_area_id` field in my local instance of the database, but that is not a valid solution for an automated migration. I like the [natural key](https://docs.djangoproject.com/en/1.10/topics/serialization/#topics-serialization-natural-keys) design that you mentioned. I will try that approach for this history -> lookupcode relation, where the natural key of a lookupcode is `('code_type', 'code')`.